breakerbp/failure_ratio.go

```diff
@@ -6,12 +6,29 @@ import (
 	"time"
 
 	"github.com/apache/thrift/lib/go/thrift"
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
 	"github.com/sony/gobreaker"
 
 	"github.com/reddit/baseplate.go/log"
 	"github.com/reddit/baseplate.go/metricsbp"
 )
 
+const (
+	nameLabel = "name"
+)
+
+var (
+	circuitBreakerLabels = []string{
+		nameLabel,
+	}
+
+	circuitBreakerClosed = promauto.NewGaugeVec(prometheus.GaugeOpts{
+		Name: "circuit_breaker_closed",
```

Comment:
wdyt about

```suggestion
		Name: "breakerbp_breaker_tripped",
```

the package name will avoid conflicts if/when we migrate to a different breaker package

https://prometheus.io/docs/practices/naming/

Reply:
I actually liked this the other way.

In Prometheus I like to make boolean values 1 for the positive thing. Like `up` is.